demos/gradio_ui.py

```diff
@@ -43,13 +56,20 @@
         outputs=output,
     )
 
-
+# Command-line interface with GPU, host, and port options
 @click.command()
 @click.option("--model_dir", required=True, help="Path to the model directory.")
-@click.option("--cpu_offload", is_flag=True, help="Whether to offload model to CPU")
-def launch(model_dir, cpu_offload):
-    configure_model(model_dir, cpu_offload)
-    demo.launch()
+@click.option("--lora_path", required=False, help="Path to the lora file.")
+@click.option("--cpu_offload", is_flag=True, help="Whether to offload model to CPU.")
+@click.option("--gpu", default=None, help="Specify the GPU to use (e.g., 'cuda:0').")
```

Comment:
It seems like the `gpu` option is used to set CUDA_VISIBLE_DEVICES, so the help text is incorrect (cuda:0 isn't valid). Could you remove the option? Users can set CUDA_VISIBLE_DEVICES manually.